common/client/src/main/java/com/bazaarvoice/emodb/client/uri/EmoUriBuilder.java

```diff
@@ -149,6 +149,12 @@ public UriBuilder uri(URI uri) {
         return this;
     }
 
+    @Override
+    public UriBuilder uri(String uriTemplate) {
+        // EMODB-MODIFICATION:  Templates are not supported, so buildFromMap is not supported
```

Comment:
nitpick, comment and exception don't correspond to method name

Reply:
updated